packages/space-opera/editor/index.html

```diff
@@ -86,50 +103,57 @@
   }
 
   me-tabs {
-    background-color: #202124;  /* GOOGLE_GREY_900 */
+    background-color: var(--expandable-section-background);
     padding: 0;
-    width: 360px;
+    width: 440px;
+    border-left: 1px solid var(--tabs-conatiner-border)
   }
 
   me-tabs a {
     color: #dadcff; /* GOOGLE_GREY_300, but.. bluer */
   }
 
   me-tabbed-panel {
-    max-height: calc(100vh - 96px);
+    max-height: calc(100vh - 45px);
     overflow-y: auto;
     display: block;
   }
 </style>
 <body>
-  <model-editor>
-    <div class="app">
+<model-editor>
+  <div class="app">
     <div class="editor-body-root">
       <div class="mvContainer">
         <model-viewer-preview id="editing_adapter">
         </model-viewer-preview>
       </div>
       <me-tabs>
-        <me-tabbed-panel icon="save">
-          <model-viewer-snippet></model-viewer-snippet>
+        <me-tabbed-panel icon="home">
+          <home-container></home-container>
+        </me-tabbed-panel>
+        <me-tabbed-panel icon="import_export">
+          <me-export-panel></me-export-panel>
         </me-tabbed-panel>
         <me-tabbed-panel icon="create">
+          <me-export-panel header="true"></me-export-panel>
           <me-transformation-controls></me-transformation-controls>
           <me-ibl-selector></me-ibl-selector>
           <me-animation-controls></me-animation-controls>
           <me-hotspot-panel></me-hotspot-panel>
           <me-poster-controls></me-poster-controls>
         </me-tabbed-panel>
-        <me-tabbed-panel icon="color_lens">
-          <me-materials-panel></me-materials-panel>
-        </me-tabbed-panel>
         <me-tabbed-panel icon="photo_camera">
+          <me-export-panel header="true"></me-export-panel>
```

Comment:
UX feedback on camera panel: 
![image](https://user-images.githubusercontent.com/1649964/99305206-4fe21c80-2808-11eb-8b64-ecf964acf9d1.png)

Reply:
Also there's some kind of async bug because I'm seeing a lot of "Waiting for camera target" when a model is loaded and functional.

Reply:
The camera target is a pre-existing bug that will take a day or two to track down. I don't know if it's worth adding to this PR. It may very well go away when I factor out the initial camera (which I was planning on doing in a separate PR)

Reply:
<img width="347" alt="Screen Shot 2020-11-18 at 5 29 33 PM" src="https://user-images.githubusercontent.com/1649964/99609898-992b9b00-29c5-11eb-9963-49f6a82bd63d.png">